cli/cmd/cluster_create.go

```diff
@@ -18,8 +17,17 @@ func (r *runners) InitClusterCreate(parent *cobra.Command) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create test clusters",
-		Long:  `Create test clusters`,
-		RunE:  r.createCluster,
+		Long: `Create test clusters.
+		
+This is a beta feature, with some known limitations:
+- K3s, Kind, kurl, helmvm and Openshift clusters only support a single node.
+- EKS clusters may report ready before the nodes are completely online.
+- kurl is the only supported distribution that can be upgraded.
```

Comment:
this line should be a limitation of the cluster upgrade command, not cluster create